### DISCUSSION TITLE
Need to reload tabs: Bug or feature

### DISCUSSION THREAD

**Discussion by markvandennieuwenhuizen** [electerm/electerm#2655] [Q&A] [answered]
Hi, I am very happy with electerm running on Macbook Air . I noticed that I have to reload my tabs after a period of inactivity. 
Is there a setting in the app that I can use to change this behaviour? 
Would appreciate help with this.

Edit: I have set Keepalive interval(ms): 1000. Still my session gets terminated
Edit 2: disconnect also applies to local terminal
Edit 3: turn display off triggers disconnect

Thanks, Mark

**Comment by markvandennieuwenhuizen** [comment, marked as answer]
I should have googled a bit more. Creating ~/.ssh/config with this contents  kept my connection alive.
Host *
    ServerAliveInterval 30
    ServerAliveCountMax 2

This issue can be closed.

**Reply by zxdong262** [reply]
Wow, would add this to faq
